docling_eval/dataset_builders/xfund_builder.py

```diff
@@ -51,6 +51,7 @@ def __init__(
         split: str = "val",  # XFUND uses "val" instead of "test"
         begin_index: int = 0,
         end_index: int = -1,
+        lang: Optional[str] = None,
```

Comment:
@samiuc I see this is a useful extension. However it would be smoother if this argument was also a list, like the internal `_supported_langs`. 
E.g. `lang: Optional[List[str]]`. The treatment in the implementation code would be simplified.